Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV SERVICE_PATH xlm_macro_deobfuscator.XLMMacroDeobfuscator
 
 # Get latest version of XLMMacroDeobfuscator library and dependencies
 RUN pip install -U https://github.com/DissectMalware/xlrd2/archive/master.zip
-RUN pip install -U https://github.com/DissectMalware/pyxlsb2/archive/master.zip
+RUN pip install -U https://github.com/cccs-jh/pyxlsb2/archive/master.zip
```

Comment:
Can you please add a comment in here indicating why you're using a forked personal repo vs the original repo

Reply:
Sure, I'll add one for the use of a fork for XLMMacroDeobfuscator as well